nss-day-cohort-60/national-parks-client
# Nav bar links working

# Description
1. Added nav bar to landing page and blogs
2. All nav bar links working
3. Changed home page to Landing page


# Steps to test

1. Fetch the md-login-modal branch and switch to it
2. Click links on nav bar and make sure they are working and are routed correctly
